app/geant-exporter/ImportProcessConverter.cc

```diff
@@ -302,14 +290,27 @@ void GeantPhysicsTableWriter::operator()(const G4ParticleDefinition& particle,
                          << ProcessTypeDemangler()(process) << ")";
     }
 
-    tree_process_->Fill();
+    return process_;
+}
+
+//---------------------------------------------------------------------------//
+/*!
+ * Remove any empty ImportProcess returned by operator().
+ */
+void ImportProcessConverter::remove_empty(std::vector<ImportProcess>& processes)
```

Comment:
Since this is only used in the one place, I think it would be better to move to the calling location. If you disagree (I understand that this idiom is a little ugly so it's nicer not to have to think about it), you should either:
- use `*` instead of mutable `&`, or
- take a `vector&&` and return a `vector` by value, so that the caller would do `vec = remove_empty(std::move(vec))`

and also make this method `static` since it doesn't require class data.